packages/amplify-e2e-tests/src/__tests__/transformer-migrations/model-migration.test.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  initJSProjectWithProfile,
+  deleteProject,
+  amplifyPush,
+  amplifyPushUpdate,
+  addFeatureFlag,
+  createRandomName,
+  addAuthWithDefault,
+} from 'amplify-e2e-core';
+import { addApiWithoutSchema, updateApiSchema, getProjectMeta } from 'amplify-e2e-core';
+import { createNewProjectDir, deleteProjectDir } from 'amplify-e2e-core';
+import AWSAppSyncClient, { AUTH_TYPE } from 'aws-appsync';
+import gql from 'graphql-tag';
+(global as any).fetch = require('node-fetch');
```

Comment:
Is there any way to do this without putting fetch on the global object? It's not the end of the world, just not ideal.

Reply:
didn't spend too much investigating this, it fails without it and we have this line in every test we use the appSyncClient